tensorflow/lite/kernels/cast_test.cc

```diff
@@ -140,6 +140,50 @@ TEST(CastOpModel, CastComplex64ToComplex64) {
            std::complex<float>(6.0f, 16.0f)}));
 }
 
+TEST(CastOpModel, CastComplex64ToUInt8) {
+  CastOpModel m({TensorType_COMPLEX64, {2, 3}}, {TensorType_UINT8, {2, 3}});
+  m.PopulateTensor<std::complex<float>>(
+      m.input(),
+      {std::complex<float>(1.0f, 11.0f), std::complex<float>(2.0f, 12.0f),
+       std::complex<float>(3.0f, 13.0f), std::complex<float>(4.0f, 14.0f),
+       std::complex<float>(5.0f, 15.0f), std::complex<float>(6.0f, 16.0f)});
+  m.Invoke();
+  EXPECT_THAT(m.ExtractVector<uint8_t>(m.output()),
+              ElementsAreArray({1, 2, 3, 4, 5, 6}));
+}
+
+TEST(CastOpModel, CastFloatToUInt8) {
```

Comment:
thanks for adding these tests. Please add some negative float cases to explicitly show the behavior.

Reply:
@talumbau , thanks for your feedback, i have updated the code as per your suggestion. Kindly check.

Regards
Amit